applications/tari_watcher/src/forker.rs

```diff
@@ -44,25 +39,19 @@ impl Forker {
         &mut self,
         config: ExecutableConfig,
         base_node_grpc_address: String,
+        vn_public_json_rpc_address: String,
+        vn_gui_http_address: String,
     ) -> anyhow::Result<Child> {
-        let mut instance = Instance::new(InstanceType::TariValidatorNode, config.clone());
-        instance.bootstrap().await?;
-
-        // verify everything is up and running
-        instance.validator_ready()?;
-
+        let instance = Instance::new(InstanceType::TariValidatorNode, config.clone());
         self.validator = Some(instance.clone());
 
-        let json_rpc_public_address = instance.port.validator.jrpc_pub_address(instance.listen_ip.unwrap());
-        let web_ui_address = instance.port.validator.web_ui_address(instance.listen_ip.unwrap());
-
         let mut command = self
             .get_command(
                 config.executable_path.unwrap(),
                 "esmeralda".to_string(), // TODO: add network to cfg
                 base_node_grpc_address,
-                json_rpc_public_address.unwrap(),
-                web_ui_address.unwrap(),
+                vn_public_json_rpc_address,
```

Comment:
The watcher doesn't need the VN JSON rpc address.

e.g. the get_command can change to spawn_validator

```rust
        let mut command = Command::new(config.executable_path.unwrap_or_else(|| Path::new("tari_validator_node"))
// Add host envs
            .envs(env::envs())
// This is technically optional, the VN will default to ~/.tari/{network}/config.toml
            .arg(format!("--config={validator_node_config_path}"))
```